### ISSUE TITLE
MSEdgeIE driver update does not work

### ISSUE THREAD

**Issue by mlipok** [Danp2/au3WebDriver#516]
### Describe the bug

MSEdgeIE driver update does not work

### How to reproduce

1. run wd_demo.au3
2. choose MSEdgeIE
3. force driver update
4. run demo

### Expected behavior

driver should update and browser should start

### Screenshots
none

### Additional context
none

### System under test
MSEdgeIE

**Comment by Danp2** [comment]
This is the issue --

- Change log for IEDriverServer was updated 3 months ago to v4.16.0 
- Last release is from 6 months ago for v4.14.0

**Comment by mlipok** [comment]
Please provide
Links to check

**Comment by Danp2** [comment]
Links come from entry in `$_WD_SupportedBrowsers`

**Comment by mlipok** [comment]
So the solution is to just wait until the appropriate version is available for download?

**Comment by Danp2** [comment]
Yes, unless someone can come up with a more reliable way to retrieve this webdriver that works with the current functionality of _WD_UpdateDriver.

**Comment by mlipok** [comment]
So, since this is beyond our scope, should we consider this issue closed for our project?

**Comment by Danp2** [comment]
I suggest leaving it open to prevent further reports on the same issue.

**Comment by Sven-Seyfert** [comment]
Is it a valid consideration to skip the support if the IE driver variant in the near future? Or do you think it's good to support that driver as long as possible? I am just curious what you both think of 😇 ?

Best regards
Sven

**Comment by mlipok** [comment]
As long as possible, as long selenium support it.

**Comment by sven-seyfert** [comment]
Maybe it's time to let IE go 😁 .

<img width="1525" height="673" alt="Image" src="https://github.com/user-attachments/assets/72407478-e0b2-4f31-a3a0-4578f502a0b7" />

**Comment by JALucena** [comment]
Checking the website that au3WebDriver uses to obtain the latest available version of IEDriverServer.exe:
[https://github.com/SeleniumHQ/selenium/blob/trunk/cpp/iedriverserver/CHANGELOG](https://github.com/SeleniumHQ/selenium/blob/trunk/cpp/iedriverserver/CHANGELOG)
It can be seen that the latest version is 4.16.0.0. This is what au3WebDriver reports in the internal function __WD_GetLatestWebdriverInfo.

However, on the Selenium download site:
[https://www.selenium.dev/downloads](https://www.selenium.dev/downloads)
The latest available version is 4.14.0.

When creating the URL for version 4.16.0 (as the internal function __WD_GetLatestWebdriverInfo does), this URL doesn't point anywhere, resulting in a "not found" error.

In my opinion, the problem stems from a discrepancy between the latest version published in the changelog by the driver maintainer and the version that is actually available.

Personally, I've been using version 4.14.0 with MS Edge, and it works fine for my needs, but I had to download it manually.

**Comment by sven-seyfert** [comment]
Hi @mlipok ,

is this Github issue still relevant? I mean would you expect to support an old IE variant "msedgeie" (IEDriverServer.exe) which is only availble on unsupported Windows operation systems?

I guess there are cases where users use the old IE, but where to draw the line regarding support?
In my opinion it would be okay to get rid of the old IE support.
Nevertheless it's up to @Danp2 , at the end.

You opened the issue in April 2025, do you depend on that in any case?

Best regards
Sven

**Comment by sven-seyfert** [comment]
🛎️ ping @mlipok
